src/main/java/com/autotune/common/utils/CommonUtils.java

```diff
@@ -319,4 +320,119 @@ public static boolean isInvalidDataSource(DataSourceInfo datasource) {
         return datasource == null || datasource.getAuthenticationConfig() == null ||
                 datasource.getAuthenticationConfig().toString().isEmpty();
     }
+
+    public static RecommendationConfigItem formatMemoryUnits(RecommendationConfigItem input) {
+        if (input == null) {
+            return new RecommendationConfigItem("Input object cannot be null");
```

Comment:
Replace all the strings here as well with constants

Reply:
Added the appropriate constants